dipy/data/fetcher.py

```diff
@@ -611,33 +635,22 @@ def read_mni_template(contrast="T2"):
     >>> T2_nifti = read_mni_template("T2") # doctest: +SKIP
     Get both files in this order:
     >>> T1_nifti, T2_nifti = read_mni_template(["T1", "T2"]) # doctest: +SKIP
-
     """
-    folder = pjoin(dipy_home, 'mni_template')
+    files, folder = fetch_mni_template()
     file_dict = {"T1":pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
```

Comment:
We're talking about IIT. This one has it's own license, which requires it
be shipped with the data, which we do.

On Thu, Aug 6, 2015 at 10:12 AM, Samuel St-Jean <notifications@github.com>
wrote:

> In dipy/data/fetcher.py
> <https://github.com/nipy/dipy/pull/681#discussion_r36439856>:
>
> >      """
> > -    folder = pjoin(dipy_home, 'mni_template')
> > +    files, folder = fetch_mni_template()
> >      file_dict = {"T1":pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
>
> Are you guys talking about the MNI template or the IIT one licensing, I am
> confused.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/681/files#r36439856>.
>